sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -162,13 +162,18 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         ASSESSMENT_INVALID_OPERATION_IN_THE_CURRENT_STATE,
       );
     }
-
+    const queueConfig = await this.queueService.getQueueConfiguration(
```

Comment:
Can we review this strategy and maybe use the `defaultJobOptions` upon queue initialization?